witchcraft-server/src/metrics/jemalloc.rs

```diff
@@ -19,4 +19,12 @@ pub fn register_metrics(metrics: &MetricRegistry) {
         let _ = epoch::advance();
         stats::allocated::read().unwrap_or(0)
     });
+    metrics.gauge("process.heap.active", move || {
+        let _ = epoch::advance();
```

Comment:
downsides from advancing the epoch more frequently?

Reply:
It does add some overhead. I think we used to have some logic that deduped the epoch advances. Let me see if I can find it.

Reply:
Messaged a link to you.

Reply:
added a simplified (?) version of that logic - since `tikv_jemalloc_ctl::stats::whatever` are cached internally didn't feel the need to actually have a structure to pull out the stats we care about, and instead just debounce the epoch advancing